lib/fog/compute/kubevirt/models/template.rb

```diff
@@ -39,7 +39,7 @@ def self.parse(object)
             :parameters       => object[:parameters],
             :description      => annotations && annotations[:description],
             :tags             => annotations && annotations[:tags]
-          }.compact
```

Comment:
This seems not to be related to the rename

Reply:
indeed, but the sample.rb fails while it is here.

Reply:
Ok so let's leave it here